### ISSUE TITLE
Error getting access tokens for AAD1 and AAD2 resources

### ISSUE THREAD

**Issue by deasems** [AzureAD/microsoft-authentication-library-for-js#1718]
## Library
- [x] `@azure/msal-browser@^2.0.0-beta.2`

## Framework
N/A

## Description
I'm testing out the latest MSAL 2.0 beta with a first party app and rand into this issue. 
My app needs to get tokens for both AAD 1.0 and AAD 2.0 resources. If I include these scopes in the `loginRedirect` call:
```ts
  msalInstance.loginRedirect({
    scopes: [
      // ...
      'https://graph.microsoft.com/User.ReadBasic.All',
      'cd34d57a-a3ef-48b1-b84b-9686f0f7c099/.default',
    ],
  });
```
The MS login UI fails with the error:
> AADSTS70011: The provided request must include a 'scope' input parameter. The provided value for the input parameter 'scope' is not valid. The scope openid profile email https://graph.microsoft.com/user.readbasic.all cd34d57a-a3ef-48b1-b84b-9686f0f7c099/.default offline_access is not valid. .default scope can't be combined with resource-specific scopes.

If I don't include one of the scopes values then login succeeds without issue. The problem then is that I can't `acquireTokenSilent` for the resource that was left out of the login call. This code:
```ts
msalInstance.loginRedirect({
    scopes: [
      // ...
      // 'https://graph.microsoft.com/User.ReadBasic.All',
      'cd34d57a-a3ef-48b1-b84b-9686f0f7c099/.default',
    ],
  });
// after handleRedirectPromise resolves
  const graphResponse = await msalInstance.acquireTokenSilent({
    scopes: ['https://graph.microsoft.com/User.ReadBasic.All'],
  });
```
Fails in my app with the error:
> ClientAuthError: no_tokens_found: No tokens were found for the given scopes, and no authorization code was passed to acquireToken. You must retrieve an authorization code before making a call to acquireToken()

It's not clear to me why this request fails  - an auth code should be available to use from the initial login, no? Am I able to supply that somehow?

How can I get access tokens for both AAD1 and AAD2 resources without multiple redirects using MSAL2.0?

## Error Message
See description.

## Security
- [ ] Is this issue security related?

## Regression
- [ ] Did this behavior work before? 
Version:

## MSAL Configuration

```ts
{
  auth: {
    clientId: '459d93de-efd4-4ced-ac1a-3134a35c8117',
    navigateToLoginRequestUrl: false,
  },
  cache: {
    cacheLocation: 'localStorage',
  },
}
```

## Reproduction steps
See code snippets in the description

## Expected behavior
Ideally I'd be able to login via redirect with just the scopes `['openid', 'profile', 'email']` and then `acquireTokenSilent` could later be used to get tokens for arbitrary resources.

In the first party app scenario I don't have to deal with the consent flow so listing resources to pre-consent to on login is not necessary.

## Browsers/Environment
- [ ] Chrome
- [ ] Firefox
- [x] Edge (only tested in Edge chromium, likely affects all browsers)
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by et1975** [comment]
the scope has to match the known application uri, which in your case is probably `api://cd34d57a-a3ef-48b1-b84b-9686f0f7c099/.default`, try that?

**Comment by deasems** [comment]
`cd34d57a-a3ef-48b1-b84b-9686f0f7c099/.default` is a valid scope that I can successfully get access tokens for when that's the only resource I attempt to get a token for.

**Comment by tnorling** [comment]
@deasems If you request a `/.default` scope [you cannot also request other scopes](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#the-default-scope). So you must break these up into separate calls. However, there is a known issue in msal-browser, that prevents silent acquisition of a token for a 2nd resource resulting in the error you experienced when you tried that. This will be fixed by #1601 . For now until that fix is merged, a workaround may be to request any additional tokens you need via an interactive flow (Popup or Redirect).

**Comment by tnorling** [comment]
@deasems `msal-browser@2.0.0-beta.3` has been released to npm and should fix the multiple resource scenario you were experiencing. Let us know if that's not the case.
